digitalocean_plugin/security.py

```diff
@@ -14,21 +14,138 @@
 # limitations under the License.
 
 import os
+import random
+
+import requests
 
 from cloudify.exceptions import NonRecoverableError
 
 
-def load_token(**_):
-    """
-    This will load a security token from a local file called token.txt
-    A token can be obtained from DigitalOcean by Registering a New Developer or Authorized Application.
-    :return: the security token, as a string
-    :raises: NonRecoverableError if token is not present
-    """
-    cwd = os.path.dirname(__file__)
-    token_path = os.path.join(cwd, 'token.txt')
-    if not os.path.isfile(token_path):
-        raise NonRecoverableError('Missing security token file "%s".' % token_path)
-    with open(token_path, 'r') as f:
-        return f.read()
+class DigitalOceanSecurity(object):
+
+    pubkey_stub = "Cloudify Key {0}"
```

Comment:
for simple strings, it would be best to use single quotes. Double quotes are better used for strings nested in another string that is surrounded by single quotes.